Documentation/Examples/PdfaShowCase/Index.rst

```diff
@@ -0,0 +1,218 @@
+                  wrap = |
+               }
+
+               docKeywords = TEXT
+               docKeywords {
+                  field = keywords
+                  wrap = |
+               }
+            }
+		}
+
+	}
+
+	plugin.tx_pdfviewhelpers.settings {
+		config {
+			class = Bithost\Pdfviewhelpers\Model\PdfaShowCase
+			jpgQuality = 80
+			fonts {
+				addTTFFont {
+					roboto {
+						path = typo3conf/ext/pdfviewhelpers/Resources/Public/Examples/FullFeatureShowCase/Roboto.ttf
+					}
+					opensans {
+						path = typo3conf/ext/pdfviewhelpers/Resources/Public/Examples/FullFeatureShowCase/OpenSans.ttf
+					}
+				}
+			}
+		}
+		document {
+			title = PDFa Show Case Title
+			subject = No Subject
+			keywords = full, feature, show, case
+			outputDestination = I
+			outputPath = pdfa.pdf
+		}
+		page {
+			margins {
+				top = 20
+				right = 15
+				bottom = 20
+				left = 15
+			}
+		}
+		generalText {
+			color = #555
+		}
+		headline {
+			fontFamily = courier
+			fontStyle = B
+		}
+		list {
+			color = #555
+			fontStyle = I
+			bulletColor = #555
+		}
+	}
+
+
+.. _pdfafeatureshowcase_php:
+
+PHP
+---
+
+A custom PHP class is needed to override the TCPDF constructor, to be able to set the pdfa parameter to true.
+
+::
+
+   <?php
+   namespace Bithost\Pdfviewhelpers\Model;
+
+   /***
+    *
+    * This file is part of the "PDF ViewHelpers" Extension for TYPO3 CMS.
+    *
+    *  (c) 2016 Markus Mächler <markus.maechler@bithost.ch>, Bithost GmbH
+    *           Esteban Marin <esteban.marin@bithost.ch>, Bithost GmbH
+    *
+    *  All rights reserved
+    *
+    *  This script is part of the TYPO3 project. The TYPO3 project is
+    *  free software; you can redistribute it and/or modify
+    *  it under the terms of the GNU General Public License as published by
+    *  the Free Software Foundation; either version 3 of the License, or
+    *  (at your option) any later version.
+    *
+    *  The GNU General Public License can be found at
+    *  http://www.gnu.org/copyleft/gpl.html.
+    *
+    *  This script is distributed in the hope that it will be useful,
+    *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+    *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    *  GNU General Public License for more details.
+    *
+    *  This copyright notice MUST APPEAR in all copies of the script!
+    ***/
+
+   use Bithost\Pdfviewhelpers\Model\TCPDF;
+
+   /**
+    * PdfaTCPDF, overrides constructor to set pdfa param as true
+    *
+    * @author Markus Mächler <markus.maechler@bithost.ch>, Esteban Marin <esteban.marin@bithost.ch>
+    */
+
+   class PdfaTCPDF extends \TCPDF {
+
+       /**
+        * @return void
+        */
+       public function __construct($orientation='P', $unit='mm', $format='A4', $unicode=true, $encoding='UTF-8', $diskcache=false, $pdfa=true) {
+           parent::__construct($orientation, $unit, $format, $unicode, $encoding, $diskcache, true);
```

Comment:
could you add a comment here, showing that the `true` parameter enables pdf/a generation?